couchexport/writers.py

```diff
@@ -221,6 +221,18 @@ def _init_table(self, table_index, table_title):
         self.tables[table_index] = sheet
         self.table_indices[table_index] = 0
 
+    def get_stripped_data(self, row):
+        """
+        A generator that returns data stripped of HTML tags
+
+        .. NOTE:: This does not convert or strip HTML entities like `&amp;`
+
+        """
+        tag = re.compile('<[^<]+?>')
```

Comment:
I'm not a fan of this half-baked HTML rendering engine. It will clobber strings like "1 < 2 and 3 > 1", for example. Seems like we may need a textual rendering mode for our reports if we want to have arbitrary strings formatted nicely in both HTML and Excel.

Had a fairly detailed discussion about this with Norman, and it sounds like proper text-mode rendering of reports is not a simple task.